src/OrganisationRegistry.UI/app/system/dashboard/dashboard.module.ts

```diff
@@ -10,8 +10,9 @@ import { SystemExceptionsModule } from './../exceptions';
 import { SystemStatsModule } from './../stats';
 import { SystemConfigurationValuesModule } from './../configurationvalues';
 
-import { DashboardRoutingModule } from './dashboard-routing.module';
+import { TerminatedInKboModule } from "../terminated-in-kbo";
```

Comment:
```suggestion
import { TerminatedInKboModule } from './../terminated-in-kbo';
```

single quotes

Reply:
Also, should we add `.module` like the other import statement?

Reply:
We can, that would make it `./../terminated-in-kbo/terminated-in-kbo.module`. Not really necessary since it's exported.

Reply:
It would make it consistent with the rest. Single quotes, ./